content/en/user-manual/animation/anim-state-graph-assets.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Animstategraph Assets
+template: usermanual-page.tmpl.html
+position: 3
+---
+
+Animstategraph assets are used to organise a set of different animation states, which are all the various ways in which a model might animation. It can be used to define each of these animation states, determine when each state should play and how states transition and therefore blend between one another.. Animstategraph assets do not store or link to any real animation assets themselves, but rather act as a template for how animation assets should be organised. Actual animation assets are linked to the animstategraphs animation states through the [Anim Component](/en/user-manual/packs/components/anim/).
+
+The system was designed so that a single animstategraph can be used on many different entities, each with their own set of animation assets. An example being an animstategraph asset which manages the animations of humanoid character locomotion. This single asset could be used on a human entity, an elf entity and a dwarf entity. Each of these entities would be able to link their own character animation assets, all the while maintaining the same animation behaviour as each other.
+
+These assets are therefore state machines for a model's animation behaviour and they control the flow of animation sequences over the lifecycle of an entity. A simple animstategraph asset used to define the behaviour of a wheel may define only two animation states; static and spinning. This asset can be defined to control when the wheel starts and stops spinning, for how long it will spin, the speed of the wheel spin and how sharply it starts / stops spinning. More advanced assets can be used to combine a multitude of animation states to create complex humanoid character animation behaviour.
```

Comment:
```suggestion
These assets are therefore state machines for a model's animation behavior and they control the flow of animation sequences over the lifecycle of an entity. A simple animstategraph asset used to define the behavior of a wheel may define only two animation states; static and spinning. This asset can be defined to control when the wheel starts and stops spinning, for how long it will spin, the speed of the wheel spin and how sharply it starts / stops spinning. More advanced assets can be used to combine a multitude of animation states to create complex humanoid character animation behavior.
```